src/puppeteer/README.md

```diff
@@ -52,6 +52,19 @@ The server provides access to two types of resources:
 - JavaScript execution
 - Basic web interaction (navigation, clicking, form filling)
 
+## Configuration to use Puppeteer Server
+Here's the Claude Desktop configuration to use the Puppeter server:
+
+```json
+{
+  "mcpServers": {
+    "puppeteer": {
+      "command": "npx",
+      "args": ["-y", "@modelcontextprotocol/server-puppeteer", "http://example.com"]
```

Comment:
I don't think puppeteer server takes a URL in the args. It's passed in by calling the exposed tool functions. 

Seems like you should drop the third `"http://example.com"` arg

Reply:
Yes updated the same, thanks for the comments @sean-lynch !